models/sound.ts

```diff
@@ -0,0 +1,50 @@
+import { list } from "@keystone-6/core";
+import type { ListConfig } from "@keystone-6/core";
+import { text, image, file, relationship } from "@keystone-6/core/fields";
+import type { Lists } from ".keystone/types";
+
+export const Sound: ListConfig<Lists.Sound.TypeInfo<any>, any> = list({
+  access: {
+    item: {
+      create: ({ session }) => session?.data.status === "active",
+      update: async ({ session, item }) => {
+        // if the owner of this sound is the same as the logged in user OR the logged in user is an admin, allow the operation
+        return (
+          (!!session?.data.id && session.data.id === item.authorId) ||
+          !!session?.data.isAdmin
+        );
+      },
+      delete: async ({ session, item }) => {
+        // if the owner of this sound is the same as the logged in user OR the logged in user is an admin, allow the operation
+        return (
+          (!!session?.data.id && session.data.id === item.authorId) ||
+          !!session?.data.isAdmin
+        );
+      },
+    },
+    operation: {
+      query: () => true,
+      create: ({ session }) => session.data.status === "active",
+      update: ({ session }) => session.data.status === "active",
+      delete: ({ session }) => session.data.status === "active",
+    },
+  },
+  fields: {
+    // The user who submitted the sound
+    author: relationship({
```

Comment:
We actually don't need Sounds to be 'owned' by users - Sounds can simply exist. They'll be manually uploaded by the site admin in Keystone's backend. So we can get rid of the User <-> Sound relationship entirely. This also helps keep the Sound model clean and simple. We do still want the relationship to a Slop, but besides that, just a few fields and we're good to go.

Reply:
This is now taken care of (along with the `access` checks referencing `item.authorId`)

Reply:
@marzahlde09 we now have 2 migrations files as part of this PR - I'm proposing we 'reset' the migration so that we only end up with 1 migration file to keep it simpler. I think to achieve this you need to delete the 2 migrations you currently have and then run `npm run dev` again to let prisma do its thing. You might also need to manually remove some entries from your database's `_prisma_migrations` table to be able to get your database to a solid state.

Reply:
I've done this before - I'll take care of that tonight!

Reply:
Should be ready to go!!